arc/src/arc_curie_pme.c

```diff
@@ -20,11 +20,12 @@ void curie_pme_begin(void)
     reg_write16(NSR, saved_nsr);
 }
 
-void curie_pme_configure(u16_t global_context,
-                         PATTERN_MATCHING_DISTANCE_MODE distance_mode,
-                         PATTERN_MATCHING_CLASSIFICATION_MODE classification_mode,
-                         u16_t min_if,
-                         u16_t max_if)
+void curie_pme_configure(
+    u16_t global_context,
+    PATTERN_MATCHING_DISTANCE_MODE distance_mode,
+    PATTERN_MATCHING_CLASSIFICATION_MODE classification_mode,
+    u16_t min_if,
+    u16_t max_if)
```

Comment:
I don't like the first arg on the next line, and I don't using a whole line per arg when others would fit. I'd even consider making shorter versions of the nasty type names, at least PATTERN_MATCHING_ -> PM_.

Was this necessary for some reason?